ui/pages/confirm-transaction/confirm-transaction.component.js

```diff
@@ -159,7 +160,16 @@ export default class ConfirmTransaction extends Component {
       transactionId &&
       (!paramsTransactionId || paramsTransactionId === transactionId);
 
-    if (isTokenMethodAction && validTransactionId) {
+    let isSendWithApprove = false;
+    if (transaction?.txParams) {
+      const {
+        type,
+        txParams: { value },
+      } = transaction;
+      isSendWithApprove =
+        type === TransactionType.tokenMethodApprove && value && value !== '0x0';
```

Comment:
[suggestion/non-blocker] I think `isSendWithApprove` would be good to add to `ui/helpers/utils/transactions.util.js`

Reply:
👍 done